#
# Submitted-By: Robert Schwebel, 2006-10-19
# Committed-By: Robert Schwebel
#
# Error:
#
# no error, at least not if you have installed libdrm on the host :)
#
# Description:
#
# This code includes host header files.
#
# State:
#
# unknown
#

Index: xorg-server-1.4/hw/xfree86/os-support/linux/Makefile.am
===================================================================
--- xorg-server-1.4.orig/hw/xfree86/os-support/linux/Makefile.am	2007-10-17 18:48:54.000000000 +0200
+++ xorg-server-1.4/hw/xfree86/os-support/linux/Makefile.am	2007-10-17 18:48:58.000000000 +0200
@@ -41,7 +41,7 @@
 
 AM_CFLAGS = -DUSESTDRES -DHAVE_SYSV_IPC $(DIX_CFLAGS) $(XORG_CFLAGS) $(PLATFORM_DEFINES)
 
-INCLUDES = $(XORG_INCS) $(PLATFORM_INCLUDES) -I/usr/include/drm # FIXME this last part is crack
+INCLUDES = $(XORG_INCS) $(PLATFORM_INCLUDES)
 
 # FIXME: These need to be added to the build
 LNX_EXTRA_SRCS = \
